### PR TITLE
remove bch completely

### PR DESCRIPTION
**Pull request by mustofa-binary**
**Changes**
- remove bch from initCryptoConfig
- remove warning.jsx and confirmation.jsx
- remove accepted_bch from client store
- remove popup bitcoin cash
- alter logic in getCashierUrl to exclude confirmedBCH
- remove bch from cryptocurrencies list

### CODE REVIEW

File: src/javascript/app/pages/cashier/deposit_withdraw.js [hunk 1]
```diff
@@ -110,32 +108,8 @@ const DepositWithdraw = (() => {
         return req;
     };
 
-    const getCashierURL = (bch_has_confirmed) => {
-        if (!/^BCH/.test(Client.get('currency')) || bch_has_confirmed || Client.get('cashier_confirmed')) {
```
Comment (negarn) [inline src/javascript/app/pages/cashier/deposit_withdraw.js]: is `cashier_confirmed` removed from everywhere too?

File: src/templates/app/cashier/deposit_withdraw.jsx [hunk 2]
```diff
@@ -33,15 +33,6 @@ const DepositWithdraw = () => (
             </p>
         </div>
 
-        <div className='invisible'>
-            <div className='gr-padding-10 gr-parent invisible' id='message_bitcoin_cash'>
-                <div className='notice-msg center-text'>
-                    <div className='gr-padding-10'>{it.L('Please note that you are currently using your [_1]Bitcoin Cash[_2] account. You can only fund your account in [_1]Bitcoin Cash[_2], and not Bitcoin.', '<a href="https://www.bitcoincash.org" target="_blank" rel="noopener noreferrer">', '</a>')}</div>
-                </div>
-            </div>
-            <iframe id='cashier_iframe' src='' frameBorder='0' width='100%' scrolling='auto' />
```
Comment (negarn) [inline src/templates/app/cashier/deposit_withdraw.jsx]: don't we want to keep the iframe?

File: src/javascript/app/pages/cashier/deposit_withdraw.js (hunk 1 above)
Comment (mustofa-binary) [reply src/javascript/app/pages/cashier/deposit_withdraw.js]: I forgot one place to set cashier_confirmed, let me fix it, thank you

File: src/templates/static/get_started/cryptocurrencies.jsx [hunk 3]
```diff
@@ -94,7 +94,6 @@ const Cryptocurrencies = () => (
                                 [{ text: 'BTC/USD' }, { text: it.L('Bitcoin vs US Dollar')      }, { text: one }, { text: hundredth }, { text: hundredth }],
                                 [{ text: 'ETH/USD' }, { text: it.L('Ethereum vs US Dollar')     }, { text: one }, { text: tenth     }, { text: hundredth }],
                                 [{ text: 'LTC/USD' }, { text: it.L('Litecoin vs US Dollar')     }, { text: one }, { text: tenth     }, { text: hundredth }],
-                                [{ text: 'BCH/USD' }, { text: it.L('Bitcoin Cash vs US Dollar') }, { text: one }, { text: hundredth }, { text: hundredth }],
```
Comment (negarn) [inline src/templates/static/get_started/cryptocurrencies.jsx]: is this for metatrader? are we removing bch in metatrader?

**Pull request merged**